department-of-veterans-affairs/caseflow
# Justin/APPEALS-46818

Resolves [Update Correspondence Assignment/Reassignment Banner Verbiage](https://jira.devops.va.gov/browse/APPEALS-46818)

# Description
Please explain the changes you made here.

## Acceptance Criteria

## Testing Plan

1. Go to [Jira Issue/Test Plan Link](https://jira.devops.va.gov/browse/JIRA-12345) or list them below
2. Log in as an inbound ops team admin user (supervisor)
3. Go to the **unassigned** tab, find an NOD correspondence, and select this checkbox
4. Find a user with no permissions from the inbound ops team dropdown and assign the correspondence to this user
5. The banner header on the next page should say `Correspondence was not assigned to {user}`, and the banner message should say `Case was not assigned to user because of NOD permissions settings.`
6. Go to the **unassigned** tab, find a non-nod correspondence, and select this checkbox
7. Select a user that has a full queue from the inbound ops team dropdown and assign the correspondence to this user. (I set this up in local by manually assigning 60 correspondences to one user first, but there may be users set up in UAT that are close to max capacity to test this)
8. The banner header on the next page should say `Correspondence was not assigned to {user}`, and the banner message should say `Case was not assigned to user because maximum capacity has been reached for user's queue.`
9. Go to the **assigned** tab, find an NOD correspondence, and select that checkbox. Repeat step 4
10. The banner header on the next page should say `Correspondence was not reassigned to {user}`, and the banner message should say `Case was not reassigned to user because of NOD permissions settings.`
11. Go to the **assigned** tab, find a non-nod correspondence, and select this checkbox. Repeat step 7.
12. The banner header on the next page should say `Correspondence was not reassigned to {user}`, and the banner message should say `Case was not reassigned to user because maximum capacity has been reached for user's queue.`

Note: Thid ticket changed the banner title and message texts for:
- Assignment NOD failures
- Reassignment NOD failures
- Assignment max capacity failures
- Reassignment max capacity failures
- Assignment sensitivity level mismatch failures (I believe Jeremy has test steps for this in UAT)
- Reassignment sensitivity level mismatch failures (Again, should be testable in UAT)


# Frontend
## User Facing Changes

 BEFORE|AFTER
Before: 
![image](https://github.com/department-of-veterans-affairs/caseflow/assets/95875751/019fbd88-9dcb-4ce3-bccb-d83f1d8fc976)

After:
![image](https://github.com/department-of-veterans-affairs/caseflow/assets/95875751/5f0c0142-04f0-4cea-be12-265c4713474b)
 ---|---
Before:
![image](https://github.com/department-of-veterans-affairs/caseflow/assets/95875751/09e54b7d-14c5-40ad-9612-2e81ffd6c8ef)
After:
![image](https://github.com/department-of-veterans-affairs/caseflow/assets/95875751/78144130-b83c-4b9f-8cb5-0cb723be6d1d)
